### PR TITLE
Justin/style comments

### PR DESCRIPTION
**Pull request by justinschreiner**
addresses more intersect style comments

### CODE REVIEW

**Review by mattlaux — CHANGES_REQUESTED** [review]
I assume this is from a change in this PR. Tough to see who I am logged in as.
Screenshot 2024-11-27 at 5 24 05 PM

**Comment by justinschreiner** [comment]
change to green checkmark in representatives

File: src/pages/index.tsx [hunk 1]
```diff
@@ -24,6 +24,7 @@ export default function Home(props: Props): JSX.Element {
   const { polls, representatives, workshops } = props;
   const session = useSession();
   useCheckAddressChange();
+  const theme = useTheme();
```
Comment (mattlaux) [inline src/pages/index.tsx:27]:
```suggestion
```

File: src/pages/index.tsx [hunk 2]
```diff
@@ -1,6 +1,6 @@
 import Head from 'next/head';
 import Link from 'next/link';
-import { Box, Typography } from '@mui/material';
+import { Box, Typography, useTheme } from '@mui/material';
```
Comment (mattlaux) [inline src/pages/index.tsx:3]:
```suggestion
import { Box, Typography } from '@mui/material';
```

**Review by mattlaux — APPROVED** [review]

**Pull request merged**